Magento_CustomerCustomAttributes/templates/customer/address/edit.phtml

```diff
@@ -183,7 +173,7 @@
                         name="region_id"
                         title="<?= /* @escapeNotVerified */ __('State/Province') ?>"
                         class="select__field select__field--native validate-select"
-                        data-mage-init='{"select": {}}'
+                        style="display:none;"
```

Comment:
can we avoid inline styles?

Reply:
I think it's better to leave it inline for region input, because magento region updater script toggle `display` in inline styles anyway